correios/client.py

```diff
@@ -167,10 +193,17 @@ def close_posting_list(self, posting_list: PostingList, posting_card: PostingCar
         xml = self._generate_xml_string(posting_list)
         tracking_codes = posting_list.get_tracking_codes()
 
-        id_ = self._auth_call("fechaPlpVariosServicos", xml,
-                              posting_list.custom_id, posting_card.number, tracking_codes)
-        posting_list.close_with_id(id_)
+        try:
+            id_ = self._auth_call("fechaPlpVariosServicos", xml,
+                                  posting_list.custom_id, posting_card.number,
+                                  tracking_codes)
+        except ClientError as exc:
+            if str(exc).startswith("A PLP não será fechada"):
```

Comment:
why this check isn't inside the `_parse_exception` method?

Reply:
Because there are specific errors for each method and doing this check here it's possible to get variable values like `tracking codes` and provide more information when raise the exception.

In `_parse_exception` method are handled generic expcetions.